scripts/birds.js

```diff
@@ -1,59 +1,161 @@
+    with its vibrant blue and black feathers and imposing presence. Each of these birds brings a unique charm and character to our zoo, making it a must-visit for bird
+    enthusiasts and nature lovers alike.`
+    document.querySelector(".section-1__title--changeContent-1").textContent = `Our`
+    document.querySelector(".section-1__title--changeContent-2").textContent = `birds`
+}
+
+const defaultSection2Content = () => {
+    birdFirstName.textContent = `Feathered`
+    birdLastName.textContent = `dream`
+
+    birdImg.classList.add("hidden")
+
+    birdDesc.textContent = `At our zoo, we treat our birds like the VIPs (Very Important Parrots) they are! Each bird gets a custom-built habitat that’s more luxurious than 
+    a five-star resort, complete with all the amenities they could ever want. Our avian team makes sure every bird is in tip-top shape, well-fed, and never bored. We even 
+    have birdie spa days with feather fluffing and beak buffing. Our goal is to make sure our feathered friends are living their best lives, and trust us, they’re loving every minute of it!`
+    readMore.classList.add("hidden")
+}
+
+const buttonsAndEvents = () => {
+    for (let birdNumber = 0; birdNumber < birds.length; birdNumber++) {
+        let animalDiv = document.createElement("div")
+        
+            animalDiv.textContent = birds[birdNumber].name
+            animalDiv.classList.add("animalMenu__element")
+            animalMenu.appendChild(animalDiv)
+            
+            let animalMenuEls = document.querySelectorAll(".animalMenu__element")
+        
+            animalMenuEls.forEach((item, birdButton) => { item.addEventListener("click", () => {
+                animalMenuEls.forEach(e => e.classList.remove("animalMenu__element--active"))
+                    item.classList.add("animalMenu__element--active");
+        
+                if (birdButton === 2) {
+                    let splitName = birds[birdButton].name.split(" ")
+        
+                    birdFirstName.textContent = splitName[0];
+                    birdLastName.textContent = `${splitName[1]} ${splitName[2]}`
+        
+                    birdLastName.classList.remove("hidden")
+                    birdFirstName.classList.add("section-2__title--active")
+        
+                } else { 
+                    birdFirstName.textContent = birds[birdButton].name;
+                    birdLastName.textContent = ``;
+        
+                    birdLastName.classList.add("hidden")
+                }
+        
+                birdImg.classList.remove("hidden")
+                birdImg.src = birds[birdButton].image
+
+                birdDesc.textContent = birds[birdButton].shortDescription
+                readMore.classList.remove("hidden")
+                readMore.textContent = `Read more...`
+
+                readMore.addEventListener("click", () => {
```

Comment:
this one needs to be outside the parent click event. right now it adds a new click-event everytime we click on it :)
to understand it. try example to do a "console.log("Hello") and click on readmore.